xla/service/gpu/nccl_api.h

```diff
@@ -179,6 +179,14 @@ class NcclApi {
                                  ReductionKind reduction_kind,
                                  NcclCommHandle comm, se::Stream* stream) = 0;
 
+  // Copy data in `send_buff` from the root GPU to the `recv_buff` on 
+  // all GPUs.
+  //
+  // https://docs.nvidia.com/deeplearning/nccl/user-guide/docs/api/colls.html#ncclbroadcast
+  static absl::Status Broadcast(se::DeviceMemoryBase send_buffer,
```

Comment:
this also has to be added to nccl_api_stub.cc

Reply:
added